filter/timestamp_range.go

```diff
@@ -68,6 +66,15 @@ func NewTimestampRange(cfg baker.FilterParams) (baker.Filter, error) {
 	return f, nil
 }
 
+func (c *TimestampRangeConfig) getTime(s string) (time.Time, error) {
+	if s == "now" {
+		return time.Now(), nil
+	}
+
+	const timeLayout = "2006-01-02 15:04:05"
+	return time.Parse(timeLayout, s)
+}
+
```

Comment:
Not sure this addition warrants a new function IMHO.
I'm not against a tiny bit of repetition, when it helps readability and doesn't hurt performance where you'd need some.

Also `getTime` has been added on the config struct, so from a readability standpoint it shouldn't be declared in-between `NewTimestampRange` and the `Stats` method.

Why not do things the other way around and call the function `setTimes(start, end string) error`?